<launch>

	<arg name="rviz" default="true" />
	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find ll_localizer)/rviz_cfg/rviz.rviz" />
	</group>

	<!-- Launch file for Livox Mid360 LiDAR -->
	<rosparam command="load" file="$(find ll_localizer)/config/mid360.yaml" />

	<param name="runtime_pos_log_enable" type="bool" value="false" />
	<node pkg="ll_localizer" type="run_localizer" name="ll_localizer" output="screen" />

</launch>